src/main/java/no/nav/familie/integrasjoner/dokarkiv/metadata/BarnetrygdVedtakMetadata.kt

```diff
@@ -0,0 +1,18 @@
+package no.nav.familie.integrasjoner.dokarkiv.metadata
+
+import no.nav.familie.integrasjoner.dokarkiv.client.domene.JournalpostType
+import org.springframework.stereotype.Component
+
+@Component
+object BarnetrygdVedtakMetadata : DokumentMetadata {
+    override val journalpostType: JournalpostType = JournalpostType.UTGAAENDE
+    override val fagsakSystem: String?= "FS36" //TODO: find out a real id
```

Comment:
maybe we should use "IT01" (Infotrygd) instead of the enum value for Foreldrepenger?

Reply:
ok